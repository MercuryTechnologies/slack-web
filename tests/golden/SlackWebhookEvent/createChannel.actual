EventEventCallback
    ( EventCallback
        { eventId = EventId
            { unEventId = "Ev0445DETPAA" }
        , teamId = TeamId
            { unTeamId = "T043DB835ML" }
        , eventTime = MkSystemTime
            { systemSeconds = 1664404606
            , systemNanoseconds = 0
            }
        , event = EventChannelCreated
            ( ChannelCreatedEvent
                { channel = CreatedChannel
                    { id = ConversationId
                        { unConversationId = "C04493BRXEZ" }
                    , isChannel = True
                    , name = "new-channel"
                    , nameNormalized = "new-channel"
                    , creator = UserId
                        { unUserId = "U043H11ES4V" }
                    , created = MkSystemTime
                        { systemSeconds = 1664404606
                        , systemNanoseconds = 0
                        }
                    , isShared = False
                    , isOrgShared = False
                    , contextTeamId = TeamId
                        { unTeamId = "T043DB835ML" }
                    }
                }
            )
        }
    )