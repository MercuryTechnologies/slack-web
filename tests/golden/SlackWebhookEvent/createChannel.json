{
  "token": "aaaa",
  "team_id": "T043DB835ML",
  "api_app_id": "A0442TUPHGR",
  "event": {
    "type": "channel_created",
    "channel": {
      "id": "C04493BRXEZ",
      "is_channel": true,
      "name": "new-channel",
      "name_normalized": "new-channel",
      "created": 1664404606,
      "creator": "U043H11ES4V",
      "is_shared": false,
      "is_org_shared": false,
      "context_team_id": "T043DB835ML"
    },
    "event_ts": "1664404606.000700"
  },
  "type": "event_callback",
  "event_id": "Ev0445DETPAA",
  "event_time": 1664404606,
  "authorizations": [
    {
      "enterprise_id": null,
      "team_id": "T043DB835ML",
      "user_id": "U0442US8QGH",
      "is_bot": true,
      "is_enterprise_install": false
    }
  ],
  "is_ext_shared_channel": false
}
